bot/seasons/easter/egg_decorating.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import random
+from io import BytesIO
+from pathlib import Path
+
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+COLOURS = [
+    (255, 0, 0, 255), (255, 128, 0, 255), (255, 255, 0, 255), (0, 255, 0, 255),
+    (0, 255, 255, 255), (0, 0, 255, 255), (255, 0, 255, 255), (128, 0, 128, 255)
+]  # Colours to be replaced - Red, Orange, Yellow, Green, Light Blue, Dark Blue, Pink, Purple
+
+IRREPLACEABLE = [
+    (0, 0, 0, 0), (0, 0, 0, 255)
+]  # Colours that are meant to stay the same - Transparent and Black
+
+
+class EggDecorating(commands.Cog):
+    """A Command that lets you decorate some easter eggs!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=["decorateegg"])
+    async def eggdecorate(self, ctx, *colours):
+        """
+        This 'paints' a beautiful egg using inputted colours
+
+        It picks from a random set of designs and alters the colours to the user's liking
+        """
+
+        if len(colours) < 2:
+            return await ctx.send("You must include at least 2 colours!")
+
+        invalid = []
+        converted = []
+        for c in colours:
```

Comment:
Can some handling be added for folks who try defining colors with comma separators? We don't necessarily want, for example, `.eggdecorate blue, blurple, green` to fail if it can be avoided.

Reply:
I don't really think the comma separation needs to be handled other than maybe mentioning what's accepted in the help output.
This is doubly so if this was altered to using a proper type-hinted converter which is going to be my recommendation coming up in the review.
If we start accepting comma separated arguments now, it'll be inconsistent with other command formats and would make us have to support it for other commands which isn't a valid, natively handled argument parsing format with the d.py commands extension.